flink-job/build.gradle

```diff
@@ -103,29 +102,29 @@ dependencies {
 
 
 
-    compile "org.apache.logging.log4j:log4j-slf4j-impl:$log4jVersion"
-    compile "org.apache.logging.log4j:log4j-api:$log4jVersion"
-    compile "org.apache.logging.log4j:log4j-core:$log4jVersion"
-    compile "org.apache.logging.log4j:log4j-jcl:$log4jVersion"
+    compile "org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}"
```

Comment:
What is the reason to use curly braces for simple variables substitution?

Reply:
imho, clearer and less error-prone in case of additions, e.g. `$log4jVersions` vs. `${log4jVersion}s`

Reply:
I do not see how it is clearer. Imho this just adds the tedious part of adding the braces when pasting in a variable.

Reply:
Yes, omitting the braces makes it easier to work with (selecting, then pasting etc), but that is not what I meant. I rather like to avoid ambiguities that may arise when omitting the braces, especially in a language I'm not home to (see [1] for details and a few examples like `$log4jVersion.x` looking for a property `x` there). I personally try to be consistent in the variable style (or here interpolation style) and not just use the braces when absolutely necessary.
The current style was a mix of these styles which we _could_ keep as well, but I proposed to unify that.

[1] http://docs.groovy-lang.org/latest/html/documentation/index.html#_string_interpolation

Reply:
Cases like "$log4jVersion.x" are easy to avoid - the whole version has to be a variable. And unification could also go the other way, but OK.